fastlane/actions/verify_keys.rb

```diff
@@ -0,0 +1,42 @@
+module Fastlane
+  module Actions
+
+    class VerifyKeysAction < Action
+      def self.run(params)
+
+        # Ensure no deploys with OSS keys
+        require 'cocoapods-core'
+        puts "Validating CocoaPods Keys"
+
+        podfile = Pod::Podfile.from_file "Podfile"
```

Comment:
Whoa, maybe! I mean, it imposes some restrictions on valid keys (they must not be a `-`) but I think that's reasonable. If I follow the instructions produced in `fastlane new_action`, that'll do it?